gravitee-apim-portal-webui-next/src/app/homepage/homepage.component.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2025 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/*
+ * Copyright (C) 2025 The Gravitee team
+ */
+import { NgIf } from '@angular/common';
+import { Component, Input } from '@angular/core';
+
+import { PageComponent } from '../../components/page/page.component';
+import { Page } from '../../entities/page/page';
+
+@Component({
+  selector: 'app-homepage',
+  standalone: true,
+  imports: [NgIf, PageComponent],
+  template: `
+    <ng-container *ngIf="homepage as page">
+      <app-page [page]="page" />
+    </ng-container>
+  `,
+})
+export class HomepageComponent {
+  @Input() homepage: Page | null = null;
+}
```

Comment:
```suggestion
@Component({
  selector: 'app-homepage',
  imports: [PageComponent],
  template: `
    @if(homepage())
      <app-page [page]="homepage()" />
      }
  `,
})
export class HomepageComponent {
  homepage = input<Page>({})
}
```
- Standalone is not necessary to specify as it is true by default
- Do not use `*ngIf`
- Use signals for the input

Reply:
done.